app/controllers/application_controller.rb

```diff
@@ -20,6 +20,19 @@ def load_site_configuration
     @site = Site.load
   end
 
+  def after_sign_in_path_for(resource)
```

Comment:
@alesrosina this should only run is the feature flag is active:

`if Site.load.public_signups_active?
`

Reply:
Please add a test (controller test recommended - have a look at MentorMatchingsControllerSpec for an example)

Reply:
Yes, valid point. Will add, thanks!